readme.md

```diff
@@ -82,6 +82,10 @@ mixpanel.people.clear_charges('billybob');
 // delete a user
 mixpanel.people.delete_user('billybob');
 
+// delete a user in Mixpanel Engage without altering $last_seen
+// - pass option `$ignore_time: true` to prevent the $last_seen property from being updated
+mixpanel.people.delete_user('billybob', {$ignore_time: true});
```

Comment:
I'm afraid I don't quite get this; can you explain what the intention is? Once a profile is deleted, it doesn't exist anymore, nor does its `last_seen` property.